src/main/resources/assets/style/main.sass

```diff
@@ -1,88 +1,11 @@
+@use 'components'
+
+// Global variables used by css (not compile time)
+:root
+	--normal-text-size: 14px
```

Comment:
This seems to be a global css change... is that safe???

Reply:
Yes css variables are safe